src/app/components/character/character.component.html

```diff
@@ -1,2 +1,2 @@
 <img id='image'src='../../assets/char_placeholder.png'>
-<h1 id='name'></h1>
+<h1 id='name'>{{character.name}}</h1>
```

Comment:
`characterName` is better because the word 'name' is used everywhere in the DOM and TS.